zubhub_frontend/zubhub/src/components/Navbar/Navbar.jsx

```diff
@@ -113,7 +113,7 @@ function PageWrapper(props) {
         } else if (searchType === SearchType.PROJECTS) {
           completions = await api.autocompleteProjects({ query, token });
           completions = completions
-          .filter(c=>( c.creator.id === props.auth.id && c.publish.type !== 1 ))
+          // .filter(c=>( c.creator.id === props.auth.id && c.publish.type !== 1 ))
```

Comment:
I will delete this line if there is no need to filter the projects and show all of them

Reply:
We should show all projects here. I think we should remove this line and test if this works fine. This could be a regressions as Raymond suggests here: https://github.com/unstructuredstudio/zubhub/issues/419#issuecomment-1806989291